docs/architecture.md

```diff
@@ -187,11 +184,11 @@ After creating a VirtualMachine it can be switched on or off like this:
 
     # Start the virtual machine:
     kubectl patch virtualmachine vm --type merge -p \
-        '{"spec":{"running":true}}'
+        '{"spec":{"runStrategy": "Always"}}'
```

Comment:
(Regarding line 183, `kubectl` can be used too:`, that I can't comment on for some reason):

I wonder if we should mention here the difference between the imperative (i.e. virtctl) and declarative (i.e. run strategies) approaches. Maybe we can add a section that says something like:

```
Virtual Machines can be turned on/off in an imperative or a declarative manner.
Setting a Run Strategy like Always or Halted means that the system will continuously
try to ensure the Virtual Machine is turned on/off. However, with the Manual strategy,
the user would imperatively choose when to turn the VM on or off, without the system
performing any automatic actions.
```

WDYT?

Reply:
Let's split it around the examples, wdyt?